pkg/volume/util/subpath/subpath_windows.go

```diff
@@ -342,6 +348,10 @@ func doSafeMakeDir(pathname string, base string, perm os.FileMode) error {
 		if stat.Mode()&os.ModeSymlink != 0 {
 			return fmt.Errorf("subpath %q is an unexpected symlink after Mkdir", currentPath)
 		}
+		// go1.23 behavior change: https://github.com/golang/go/issues/63703#issuecomment-2535941458
+		if stat.Mode()&os.ModeIrregular != 0 {
+			return fmt.Errorf("subpath %q is an unexpected symlink after Mkdir", currentPath)
```

Comment:
update error message to say "irregular file"

Reply:
fixed